packages/design-system/src/components/Table/TableBody.test.jsx

```diff
@@ -6,35 +6,36 @@ import TableRow from './TableRow';
 import { mount } from 'enzyme';
 
 const defaultTableBodyChildren = (
-  <TableRow key="1">
+  <TableRow>
     <TableCell>Cell a</TableCell>
     <TableCell>Cell a</TableCell>
   </TableRow>
 );
-const defaultTableBodyProps = {
-  className: 'foo-body',
-};
 
 function render(customProps = {}, children) {
   const props = Object.assign({}, customProps);
 
   if (!children) {
-    children = <TableBody {...defaultTableBodyProps}>{defaultTableBodyChildren}</TableBody>;
+    children = <TableBody {...props}>{defaultTableBodyChildren}</TableBody>;
   }
-
   return {
     props: props,
-    wrapper: mount(<Table {...props}>{children}</Table>),
+    wrapper: mount(<Table>{children}</Table>),
```

Comment:
Why are you rendering a <Table> in this test file? Generally if you are testing `TableBody` you will just be testing that component in isolation